db/migrate/20181105061336_cast_key_types_and_values_in_parameters.rb

```diff
@@ -1,46 +0,0 @@
-class CastKeyTypesAndValuesInParameters < ActiveRecord::Migration[5.2]
```

Comment:
IIRC you need to keep the migration as an empty class or rails might get confused if it already ran.

Reply:
Yeah. I think along with removal of this migration file I will need to delete entry from  `schema_migrations` so that it won't cause any issue. Will that work? Or should I keep empty class?

Reply:
Done! Kept an empty class.